paper/paper.md

```diff
@@ -11,36 +11,57 @@ authors:
 
-ERAD is a free, open-source Python toolkit for computing equity and resilience measures in the face of hazards like earthquakes and flooding. It uses graph database to store data and perform computation at the household level for a variety of critical services that are connected by power distribution network. It uses asset fragility curves, which are functions that relate hazard severity to survival probability for power system assets including cables, transformers, substations, roof-mounted solar panels, etc. recommended in top literature. Programs like undergrounding, microgrid, and electricity backup units for critical infrastructures may all be evaluated using metrics and compared across different neighborhoods to assess their effects on equity and resilience.
+# Statement of need
 
-ERAD is designed to be used by researchers, students, community stakeholders, distribution utilities to understand and possibly evaluate effectiveness of different post disaster programs to improve resilience and equity. It was funded by National Renewable Energy Laboratory (NREL) and made publicy available with open license.
+~~In the last few years, we have seen a surge in natural disaster events affecting the power grid, in part due to increased frequency and intensity of extreme weather due to climate change. Similarly, human-driven malicious acts, such as cybersecurity attacks can have similar, wide-spread impacts on the grid. Grid outages in turn effect other services, which have a direct impact on people's lives during an emergency.~~
 
-# Statement of need
+Much of the past work in evaluating equity and resilience have looked at historic data. For instance *<u>(XXX need to add a sentence for each of a 2+ of additional pieces of work, a place to start might be reviewing the citations in [@lin_equity-based_2022])</u>* In another example Brockway, et al. [@brockway_inequitable_2021] show how existing grid infrastructure limits may limit disadvantage community ability to charge electric vehicles and their access to DERs by looking at the geospatial intersection of demographics and grid hosting capacity data. ERAD instead provides a way to look ahead at possible future outages and service access challenges.
```

Comment:
@KapilDuwadi, @AadilLatif and/or Kwami: not sure if this is the right framing. If it is we need a few more citations. If not it would be good to reframe--though still need a few citations to support.